packages/create-hydrogen/src/services/init.ts

```diff
@@ -164,6 +164,16 @@ async function init(options: InitOptions) {
           task.title = 'Completed clean up'
         },
       },
+      {
+        title: 'Initializing a Git repository...',
+        task: async (_, task) => {
+          await file.move(`${templateScaffoldDir}/_gitignore`, `${templateScaffoldDir}/.gitignore`)
```

Comment:
I was under the impression that we were using a consistent template here? Is there a reason we can't create the gitignore? If not, we should remove this line because it is duplicating functionality from the hydrogen repo.

https://github.com/Shopify/hydrogen/blob/v1.x-2022-07/scripts/compile-template.mjs#L82